src/cli/setup_mode.rs

```diff
@@ -1,3 +1,5 @@
+use std::collections::HashMap;
```

Comment:
Finally, at Rustup it is recommended to use atomic commits with informative commit messages, please have a look at the examples on the `master` branch.